src/sentry/static/sentry/app/components/dropdownButton.tsx

```diff
@@ -32,17 +32,15 @@ const DropdownButton = ({
   <StyledButton type="button" isOpen={isOpen} ref={forwardedRef} {...props}>
     {prefix && <LabelText>{prefix}:</LabelText>}
     {children}
-    {showChevron && <StyledChevronDown />}
+    {showChevron && <StyledChevronDown size="10px" direction={isOpen ? 'up' : 'down'} />}
   </StyledButton>
 );
 
 DropdownButton.defaultProps = {
   showChevron: true,
 };
 
-const StyledChevronDown = styled(props => (
-  <InlineSvg src="icon-chevron-down" {...props} />
-))`
+const StyledChevronDown = styled(IconChevron)`
```

Comment:
Looks like this can be just `StyledChevron` now

```suggestion
const StyledChevron = styled(IconChevron)`
```